package_adafruit_index.json

```diff
@@ -3,6 +3,54 @@
     {
       "websiteURL": "https://adafruit.com",
       "tools": [
+        {
```

Comment:
link to most platforms are duplicated from https://downloads.arduino.cc/packages/package_index.json

```json
        {
          "name": "bossac",
          "version": "1.9.1-arduino2",
          "systems": [
            {
              "host": "i686-mingw32",
              "url": "http://downloads.arduino.cc/tools/bossac-1.9.1-arduino2-windows.tar.gz",
              "archiveFileName": "bossac-1.9.1-arduino2-windows.tar.gz",
              "checksum": "SHA-256:5c994d04354f0db8e4bea136f49866d2ba537f0af74b2e78026f2d4fc75e3e39",
              "size": "1260628"
            },
            {
              "host": "x86_64-apple-darwin",
              "url": "http://downloads.arduino.cc/tools/bossac-1.9.1-arduino2-osx.tar.gz",
              "archiveFileName": "bossac-1.9.1-arduino2-osx.tar.gz",
              "checksum": "SHA-256:b7732129364a378676604db6579c9b8dab50dd965fb50d7a3afff1839c97ff80",
              "size": "47870"
            },
            {
              "host": "x86_64-pc-linux-gnu",
              "url": "http://downloads.arduino.cc/tools/bossac-1.9.1-arduino2-linux64.tar.gz",
              "archiveFileName": "bossac-1.9.1-arduino2-linux64.tar.gz",
              "checksum": "SHA-256:9eb549874391521999cee13dc823a2cfc8866b8246945339a281808d99c72d2c",
              "size": "399532"
            },
            {
              "host": "i686-pc-linux-gnu",
              "url": "http://downloads.arduino.cc/tools/bossac-1.9.1-arduino2-linux32.tar.gz",
              "archiveFileName": "bossac-1.9.1-arduino2-linux32.tar.gz",
              "checksum": "SHA-256:10d69f53f169f25afee2dd583dfd9dc803c10543e6c5260d106725cb0d174900",
              "size": "384951"
            },
            {
              "host": "arm-linux-gnueabihf",
              "url": "http://downloads.arduino.cc/tools/bossac-1.9.1-arduino2-linuxarm.tar.gz",
              "archiveFileName": "bossac-1.9.1-arduino2-linuxarm.tar.gz",
              "checksum": "SHA-256:c9539d161d23231b5beb1d09a71829744216c7f5bc2857a491999c3e567f5b19",
              "size": "361915"
            },
            {
              "host": "aarch64-linux-gnu",
              "url": "http://downloads.arduino.cc/tools/bossac-1.9.1-arduino2-linuxaarch64.tar.gz",
              "archiveFileName": "bossac-1.9.1-arduino2-linuxaarch64.tar.gz",
              "checksum": "SHA-256:c167fa0ea223966f4d21f5592da3888bcbfbae385be6c5c4e41f8abff35f5cb1",
              "size": "442853"
            }
          ]
        },
```